### PR TITLE
rook: update rook to 1.4.6

### PR DESCRIPTION
**Pull request by ipochi**
@surajssd Can you please review if I have updated the manifests `pkg/components/rook-ceph/manifests.go` correctly.

Signed-off-by: Imran Pochi <imran@kinvolk.io>

### CODE REVIEW

**Review by invidian — APPROVED** [review]
Looks OK

File: pkg/components/rook-ceph/manifests.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ metadata:
   namespace: {{ .Namespace }}
 spec:
   cephVersion:
-    image: ceph/ceph:v15.2.4-20200630
```
Comment (surajssd) [inline pkg/components/rook-ceph/manifests.go:28]: Generally ceph pushes image to same tag. So to make things predictable we should tag it to the specific version that rook docs suggests.

Comment (ipochi) [reply pkg/components/rook-ceph/manifests.go:28]: Ok, I took this as an example:
https://github.com/rook/rook/blob/v1.4.6/cluster/examples/kubernetes/ceph/cluster.yaml

Comment (surajssd) [reply pkg/components/rook-ceph/manifests.go:28]: Yes the same doc also talks about the specific tag.

Also the current version of PR worked I have tested it. But please make the tag specific.

**Review by invidian — APPROVED** [review]

**Review by surajssd — APPROVED** [review]

**Pull request merged**